orc8r/gateway/python/magma/state/state_replicator.py

```diff
@@ -37,28 +37,51 @@
     SyncStatesRequest,
 )
 
-# TODO: Make DEFAULT_SYNC_INTERVAL an mconfig parameter
 DEFAULT_SYNC_INTERVAL = 60
 DEFAULT_GRPC_TIMEOUT = 10
 GARBAGE_COLLECTION_ITERATION_INTERVAL = 2
 
 
+def _resolve_sync_interval(service: MagmaService) -> int:
+    # TODO(#8806): update this once we confirm partners no longer set sync_interval service configs.
+    # Sync_interval is in seconds
+    mconfig_sync_interval = service.mconfig.sync_interval
+    if mconfig_sync_interval is None or mconfig_sync_interval <= 0:
+        logging.info(
+            "mconfig_sync_interval is none, resetting to default interval %s",
```

Comment:
this message is not correct when `mconfig_sync_interval <= 0`

Reply:
good catch updating